Dockerfile

```
@@ -0,0 +1,14 @@
+FROM alpine:latest
+
+LABEL org.opencontainers.image.authors="Adfinis AG <https://adfinis.com>"
```

Maybe something like this?
```suggestion
LABEL org.opencontainers.image.title="alpine-debug-container" \
      org.opencontainers.image.description="Alpine based debug container with tools for debugging networking issues and more." \
      org.opencontainers.image.authors="Adfinis AG <https://adfinis.com>" \
      org.opencontainers.image.vendor="Adfinis" \
      org.opencontainers.image.url="https://github.com/adfinis-sygroup/debug-container" \
      org.opencontainers.image.licenses="AGPL-3.0-or-lateræ
```
Most of what's left in https://github.com/opencontainers/image-spec/blob/master/annotations.md is stuff that we can only set at build time.